content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -5,38 +5,35 @@ weight: 60
 keywords: [kubernetes,multicluster]
 ---
 
```

Comment:
Should we change the title? I try to discourage "Istio" in page titles, since it's somewhat redundant.

For the description, should that be "Installing Istio..."?

Reply:
I can definitely do that.